Templace_I2C/Template_I2C_Basic.ino

```diff
@@ -0,0 +1,29 @@
+#include <Wire.h>
+
+int Address = 8;
+
+void setup() {
+  Wire.begin(Address);           // join i2c bus with address #8
+  Wire.onRequest(requestEvent); // register event
+}
+
+void loop() {
+  delay(100);
+}
+
+// function that executes whenever data is requested by master
+// this function is registered as an event, see setup()
+void requestEvent() {
+  union floatToBytes {
+    char buffer[4];
+    float voltageReading;
```

Comment:
rename voltage reading to something more general. 
For example, if it's a GPS, we'll have a digital input that tells us latitude and longitude. That's not a voltage reading.
Also, some sensors won't have just one float input. It might be several floats: eg. lat, long; x, y, z acceleration, etc.
This union data structure needs to be larger and more general.